opm/simulators/flow/FlowProblemComp.hpp

```diff
@@ -492,8 +589,13 @@ class FlowProblemComp : public FlowProblem<TypeTag>
         throw std::logic_error("polymer is disabled for compositional modeling and you're trying to add polymer to BC");
     }
 
+    FlowThresholdPressure<TypeTag> thresholdPressures_;
+
     std::vector<InitialFluidState> initialFluidStates_;
 
+    bool enableEclOutput_;
```

Comment:
Please always initialise `bool`s&ndash;typically to `false`.

Reply:
fixed.

Reply:
> fixed.

Thanks.  I think we should initialise `enableEclOutput_` as well, not just `enableVtkOutput_`.

Reply:
It was a mistake. I will initialize both now.